docs/HACKING.md

```diff
@@ -7,6 +7,8 @@
 - The operator is written using operator-sdk 
```

Comment:
First line of the commit message is the subject and must be no longer than 50 characters. See [format of the commit message.](https://github.com/openshift/windows-machine-config-operator/blob/master/CONTRIBUTION.md#format-of-the-commit-message)